.travis.yml

```diff
@@ -12,7 +12,6 @@ addons:
   google-chrome: latest
 
 install:
-  - npm install npm@5 -g
```

Comment:
This change should be requested to Skeleton and then applied directly to all elements, right? Currently we use npm in some elements, yarn in some...